<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editMonsterZoneFormClass</class>
 <widget class="QDialog" name="editMonsterZoneFormClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>569</width>
    <height>454</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>433</width>
    <height>160</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Edit monster zone</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout_3">
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Color :</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QComboBox" name="comboBoxColor"/>
       </item>
       <item>
        <widget class="QLabel" name="labelColor">
         <property name="minimumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Name : </string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEditName"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Ratio to encounter
a monster :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Monsters :</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTableWidget" name="tableWidgetMonsters">
         <property name="columnCount">
          <number>2</number>
         </property>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <column/>
         <column/>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="pushButtonAddMonster">
           <property name="text">
            <string>&amp;Add</string>
           </property>
           <property name="icon">
            <iconset resource="../res/global.qrc">
             <normaloff>:/commonEditor.UI/res/gtk-add.svg</normaloff>:/commonEditor.UI/res/gtk-add.svg</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonEditMonster">
           <property name="text">
            <string>&amp;Edit</string>
           </property>
           <property name="icon">
            <iconset resource="../res/global.qrc">
             <normaloff>:/commonEditor.UI/res/gtk-edit.svg</normaloff>:/commonEditor.UI/res/gtk-edit.svg</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonDeleteMonster">
           <property name="text">
            <string>&amp;Delete</string>
           </property>
           <property name="icon">
            <iconset resource="../res/global.qrc">
             <normaloff>:/commonEditor.UI/res/gtk-delete.svg</normaloff>:/commonEditor.UI/res/gtk-delete.svg</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QSpinBox" name="spinBoxRatioChance"/>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>chance on </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBoxRatioTotal"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonOK">
       <property name="text">
        <string>&amp;OK</string>
       </property>
       <property name="icon">
        <iconset resource="../res/global.qrc">
         <normaloff>:/commonEditor.UI/res/dialog-ok.svg</normaloff>:/commonEditor.UI/res/dialog-ok.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCancel">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
       <property name="icon">
        <iconset resource="../res/global.qrc">
         <normaloff>:/commonEditor.UI/res/dialog-close.svg</normaloff>:/commonEditor.UI/res/dialog-close.svg</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>comboBoxColor</tabstop>
  <tabstop>lineEditName</tabstop>
  <tabstop>spinBoxRatioChance</tabstop>
  <tabstop>spinBoxRatioTotal</tabstop>
  <tabstop>tableWidgetMonsters</tabstop>
  <tabstop>pushButtonAddMonster</tabstop>
  <tabstop>pushButtonEditMonster</tabstop>
  <tabstop>pushButtonDeleteMonster</tabstop>
  <tabstop>pushButtonOK</tabstop>
  <tabstop>pushButtonCancel</tabstop>
 </tabstops>
 <resources>
  <include location="../res/global.qrc"/>
 </resources>
 <connections/>
</ui>
